en/docs/reference/connectors/microsoft-azure-storage-connector/2.x/microsoft-azure-storage-connector-example.md

```diff
@@ -17,109 +17,113 @@ For more information about these operations, please refer to the [Microsoft Azur
 
 > **Note**: Before invoking the API, you need to create a **Storage Account** in **Microsoft Azure Storage account**. See [Azure Storage Configuration]({{base_path}}/reference/connectors/microsoft-azure-storage-connector/microsoft-azure-storage-configuration/) documentation for more information.
 
-## Configure the connector in WSO2 Integration Studio
+## Setup the Integration Project
```

Comment:
```suggestion
## Set up the Integration project
```

Reply:
Also use sentence case for titles. Check and fix all places. https://learn.microsoft.com/en-us/style-guide/text-formatting/formatting-titles